website/content/docs/k8s/helm.mdx

```diff
@@ -1784,6 +1787,9 @@ Use these links to navigate to a particular top-level stanza.
     or may not be broadly accessible depending on your Kubernetes cluster.
     Set this to false to skip syncing ClusterIP services.
 
+  - `syncLoadBalancerEndpoints` ((#v-synccatalog-syncloadbalancerendpoints)) (`boolean: false`) - If true, LoadBalancer service endpoints instead of ingress addresses will be synced to Consul. 
+    If false, LoadBalancer endpoints are not synced to Consul.
```

Comment:
```suggestion
  - `syncLoadBalancerEndpoints` ((#v-synccatalog-syncloadbalancerendpoints)) (`boolean: false`) - If true, LoadBalancer service endpoints instead of ingress addresses are synced to Consul. 
    If false, LoadBalancer endpoints are not synced to Consul.
```
Nit: present tense for both options